packages/babel-plugin-transform-pipeline-operator/test/fixtures/pipeline-operator/currying/actual.js

```diff
@@ -0,0 +1,6 @@
+
+var map = (fn) => (array) => array.map(fn);
+
+var result = [10,20] |> map(x => x * 20);
```

Comment:
(clarification) is the `map(x => x * 20)` call to create the actually applied function supposed to not happen until after the LHS of `|>` is evaluated?

Reply:
Per the [proposed spec](https://github.com/tc39/proposal-pipeline-operator/pull/51/files#diff-3540caefa502006d8a33cb1385720803R18), it has to be after.

Reply:
So it works similarly to `&&` and `||` as opposed to a regular math operator.

Reply:
Yah, this definitely isn't a compose.